Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.yml

```diff
@@ -847,14 +854,13 @@ script:
     - contextPath: ASM.ExternalWebsite
       description: A list of the websites results assets.
     description: Get external websites assets.
-  dockerimage: demisto/python3:3.10.13.87159
+  dockerimage: demisto/python3:3.10.13.89009
   isFetchSamples: true
   isfetch: true
   script: ''
   subtype: python3
   type: python
-defaultmapperin: Xpanse - Incoming Mapper
-defaultclassifier: Xpanse - Classifier
```

Comment:
why did you remove them?

Reply:
That was unintentional. I can resubmit without those changes if needed.

Reply:
just revert them